setup.py

```diff
@@ -18,7 +18,8 @@
                       'numpy',
                       'pandas',
                       'pypdb',
-                      'matplotlib'],  # Install requirements extracted from requirements.txt
+                      'matplotlib',
+                      'scipy'],  # Install requirements extracted from requirements.txt
```

Comment:
Add pandda_gemmi - it'll pick up that the user hasn't installed it, I think? Give it a go.